app/src/main/res/layout/activity_main.xml

```diff
@@ -1,83 +1,95 @@
-    >
+<layout>
 
-    <android.support.design.widget.TextInputLayout
-        android:id="@+id/txtInputLayoutKeyword"
-        android:layout_width="@dimen/dp_368"
-        android:layout_height="wrap_content"
-        app:errorTextAppearance="@style/ErrorText"
-        app:hintTextAppearance="@style/LabelText"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.196"
-        >
+    <data>
 
-        <EditText
-            android:id="@+id/edtKeyword"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:hint="@string/keyword"
+        <variable
+            name="viewModel"
+            type="com.fstyle.structure_android.screen.main.MainViewModel"
             />
-    </android.support.design.widget.TextInputLayout>
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/search_user_github"
-        android:textSize="@dimen/sp_23"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintHorizontal_bias="0.455"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.032"
-        tools:text="@string/app_name"
-        />
 
-    <Button
-        android:layout_width="100dp"
-        android:layout_height="60dp"
-        android:onClick="onSearchButtonClicked"
-        android:text="@string/search"
-        android:textAllCaps="false"
-        android:textSize="@dimen/dp_16"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.578"
-        tools:text="Submit"
-        style="@style/Widget.AppCompat.Button.Colored"
-        />
+    </data>
 
-    <android.support.design.widget.TextInputLayout
-        android:id="@+id/txtInputLayoutNumberLimit"
-        android:layout_width="@dimen/dp_368"
-        android:layout_height="wrap_content"
-        app:errorTextAppearance="@style/ErrorText"
-        app:hintTextAppearance="@style/LabelText"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.35"
+    <android.support.constraint.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
ERROR: [MissingPrefix] Unexpected namespace prefix "xmlns" found for tag `android.support.constraint.ConstraintLayout`
 Most Android views have attributes in the Android namespace. When referencing these attributes you *must* include the namespace prefix, or your attribute will be interpreted by `aapt` as just a custom attribute.

Similarly, in manifest files, nearly all attributes should be in the `android:` namespace.